src/app/secondary-components/publications/publications.component.html

```diff
@@ -26,6 +26,11 @@ <h1 translate>publi.title</h1>
                         <fa-icon [icon]="['fab', 'researchgate']"></fa-icon>
                     </a>
                 </div>
+        </div>
+    </div>
+    <div *ngIf="notFound" class="row text-center mt-5">
+        <h2 translate>publi.not-found</h2>
```

Comment:
Add `publi.not-found` description at `src/assets/i18n/en.json`in english and at `src/assets/i18n/pt.json` in brazilian portuguese.